src/tactic/ui/table/format_element_wdg.py

```diff
@@ -547,6 +547,9 @@ def get_format_value(my, value, format):
             div.add_class("spt_boolean_top")
             from pyasm.widget import CheckboxWdg
             checkbox = CheckboxWdg(my.get_name())
+            if my.attributes.get('edit') == 'false':
```

Comment:
should this also include False or "False"? This attribute could be passed from both python or javascript right?